tiger/tth.go

```diff
@@ -28,6 +28,32 @@ func TreeLeaves(r io.Reader) (lvl Leaves, err error) {
 	return
 }
 
+// ToTreeHash converts leaves into a hash.
+func (in Leaves) ToTreeHash() Hash {
```

Comment:
Maybe call it `TreeHash`?

Reply:
ok, at the moment i don't care about naming, we'll get the clear picture when the library is complete.